gwi-project/src/components/Dropdown/index.tsx

```diff
@@ -0,0 +1,90 @@
+import { css } from "@emotion/css"
+import { capitalizeFirstLetter } from "../../Utils/stringUtils"
+
+export function Dropdown(props: {
+  allBreeds: string[]
+  setBreed: (arg0: string) => void
+}) {
+  return (
+    <div className={styles.overview}>
+      <div className={styles.container}>
+        <div className={styles.scrollbar}>
+          {props.allBreeds.map((breed: string) => (
+            <div
+              onClick={() => props.setBreed(breed)}
+              key={breed}
+              className={styles.productContainer}
+            >
+              {capitalizeFirstLetter(breed)}
+            </div>
+          ))}
+        </div>
+      </div>
+    </div>
+  )
+}
+
+const styles = {
+  overview: css`
+    position: absolute;
+    right: 0;
+    max-height: 100%;
+    transform: translateY(10px);
+    cursor: default;
+  `,
+  container: css`
+    display: flex;
+    border-radius: 5px;
+    flex-direction: column;
+    border: 2px solid var(--lightgray);
+    background-color: var(--white);
+    max-height: 500px;
+    overflow: hidden;
+    padding: 10px;
+  `,
+  productContainer: css`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    padding: 26px;
+    border: 2px solid var(--lightgray);
+    border-radius: 5px;
+    width: 150px;
+    font-size: 16px;
+    cursor: pointer;
+
+    :hover {
+      font-weight: bold;
+      border: 2px solid var(--gray);
+    }
+  `,
+  scrollbar: css`
+    overflow-y: auto;
+    overflow-x: hidden;
+    border-radius: 10px;
+    ::-webkit-scrollbar {
+      width: 10px;
+      border-radius: 10px;
+    }
+    ::-webkit-scrollbar-track {
+      background: var(--tooltip-track);
+      border-radius: 10px;
+    }
+    ::-webkit-scrollbar-thumb {
+      background: var(--lightgray);
+      border-radius: 10px;
+    }
+    ::-webkit-scrollbar-thumb:hover {
+      background: var(--gray);
+      border-radius: 10px;
+    }
```

Comment:
Does this work on browsers that don't work with webkit? if not what the solution would be there?

Reply:
Only internet explorer does not support WebKit so I can not test it. I have tested it with Chrome, Safari, Firefox, Brave and it works perfectly.

Reply:
And how would you fix it in internet explorer for instance?

Reply:
I would try something like    
            scrollbar-3dlight-color: #4fa5dd;
            scrollbar-arrow-color: #ffcc99;
            scrollbar-darkshadow-color: #000000;
            scrollbar-face-color: #99cccc;
            scrollbar-highlight-color: #ffffcc;
            scrollbar-shadow-color: #000033;
            scrollbar-track-color: #cfe5f5;
            
            I am not sure about the colors (if they match a lot) it is just an example.

Reply:
awesome mate! many many thx :)

Reply:
Thank you for the feedback.
Do you know when you will get back to me? 😄